cns/hnsclient/hnsclient_windows.go

```diff
@@ -301,10 +305,10 @@ func createHostNCApipaNetwork(
 		if interfaceExists, _ := networkcontainers.InterfaceExists(hostNCLoopbackAdapterName); !interfaceExists {
 			ipconfig := cns.IPConfiguration{
 				IPSubnet: cns.IPSubnet{
-					IPAddress:    localIPConfiguration.GatewayIPAddress,
+					IPAddress:    hnsLoopbackAdapterIPAddress,
 					PrefixLength: localIPConfiguration.IPSubnet.PrefixLength,
 				},
-				GatewayIPAddress: localIPConfiguration.GatewayIPAddress,
+				GatewayIPAddress: hnsLoopbackAdapterIPAddress,
```

Comment:
why changing here? anyway you are updating with `updateGwForLocalIPConfiguration`

Reply:
we should always use .1 for this ip

Reply:
loopback adapter gw should always be .1. we dont want to change it